### PR TITLE
Fix: File Browser roles check boxes are now synchronized with stored …

### PR DESCRIPTION
**Pull request by mean2me**
…settings

### CODE REVIEW

File: CKEditorOptions.ascx.cs [hunk 1]
```diff
@@ -394,6 +390,10 @@ protected void Page_Load(object sender, EventArgs e)
 
                 FillFolders();
 
+                BindUserGroupsGridView();
+
+                BindOptionsData();
```
Comment (tpluscode) [inline CKEditorOptions.ascx.cs]: @mean2me I think you did the same change before, moving these two calls. Didn't this change cause issues before?

File: CKEditorOptions.ascx.cs [hunk 2]
```diff
@@ -1118,18 +1118,38 @@ private void FillRoles()
...
-                    roleItem.Enabled = false;
+                    if (!string.IsNullOrEmpty(roleId) && !string.IsNullOrWhiteSpace(roleId))
+                    {
+                        activeRoles.Add(int.Parse(roleId.Trim()));
+                    }
                 }
-
-                chblBrowsGr.Items.Add(roleItem);
             }
+            return activeRoles;
```
Comment (tpluscode) [inline CKEditorOptions.ascx.cs]: Actually I would actually change this method to be implemented as generator

```diff
-private List<int> GetActiveRolesIds()
+private IEnumerable<int> GetActiveRolesIds()
// ...
-activeRoles.Add(int.Parse(roleId.Trim()));
+yield return int.Parse(roleId.Trim());
// ...
-return activeRoles;
```

**Review by tpluscode — CHANGES_REQUESTED** [review]

File: CKEditorOptions.ascx.cs (hunk 1 above)
Comment (mean2me) [reply CKEditorOptions.ascx.cs]: @tpluscode I couldn't see any impact after recent changes in CKEditor.

**Review by tpluscode — APPROVED** [review]
Can anyone from @dnnsoftware/tag merge this?

**Review by ohine — APPROVED** [review]

**Pull request merged**